ProjectTraning/Bicycle.cs

```diff
@@ -0,0 +1,34 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace ProjectTraning
+{
+    class Bicycle: Vehicles
+    {
+        public BicycleType Type { get; set; }
```

Comment:
thinks these properties should have private setters since nobody should have a possibility to change it.